google/resource_compute_instance.go

```diff
@@ -1458,6 +1548,51 @@ func resourceComputeInstanceUpdate(d *schema.ResourceData, meta interface{}) err
 	return resourceComputeInstanceRead(d, meta)
 }
 
+func startInstanceOperation(d *schema.ResourceData, config *Config) (*compute.Operation, error) {
+	project, err := getProject(d, config)
```

Comment:
When I started implementing this, I did not include lines 1552-1567 because retrieving the project, zone and instance was already done at the beginning of `resourceComputeInstanceUpdate`. So I decided to pass `project`, `zone` and `instance` as parameters of `startInstanceOperation`, but then I felt like having all those parameters was "too much", because we can retrieve them using `d` and `config` (what I did here). Tell me what you think and what you prefer :
- passing `project`, `zone` and `instance` as parameters of `startInstanceOperation`
- keep `startInstanceOperation` like this (it duplicates code)
- or something else :smile: